docs/developers/optimistic-oracle-integration.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Integrating with the Optimistic Oracle
+sidebar_label: Integrating with the Optimistic Oracle
+---
+
+## "Deposit Box" Example Contract
```

Comment:
nit formatting: I'd like to avoid having double headers on all of our pages. I think it would be better if there was an introductory paragraph with no header, or the "Deposit Box" header was just removed.